pages/index.js

```diff
@@ -14,26 +16,31 @@ import Venue from '../app/components/venue';
 import ScrollBtn from '../app/components/scroll-btn';
 import ContactForm from '../app/components/contactForm';
 import PhotoGallery from '../app/components/photoGallery';
-import store from '../app/data/2019';
-import 'what-input';
+import AfterMovie from '../app/components/afterMovie';
 
-const buildComponent = (currentStore, year = '2019') => {
+import store from '../app/data/2020';
+import store2019 from '../app/data/2019';
+
+const buildComponent = (currentStore, prevStore, year = '2020') => {
   const baseUrl = `static/img/${year}`;
   const { speakers, sponsors, partners, communities, schedule } = currentStore;
 
+  const showMission = false;
+
   const Component = () => (
     <Layout>
       <Header />
+      <AfterMovie {...{ year: 2019, videos: prevStore.videos }} />
```

Comment:
shouldn't the year (2019) be enough for the component to display the aftermovie and the videos? this way you don't have to send the `prevStore` to this component

Reply:
right now the aftermovie component was designed as a presentation component, and all the props are being passed on from outside (year and videos). Yes year would be enough if we add a bit of logic in the component, but we open an pandora box, because other components have the same ... logic (EG: Speakers, Schedule, Sponsors, Partners, Communities). But I will pass only the year for now, as it encapsulates the logic better.